### PR TITLE
add static helper for getAbsMaxCp

### PR DESCRIPTION
**Pull request by ramarro123**
helper static function to return maxAbsCp based on pokemonID

can be used to determine the maxAbsCp of encounterer pokemon

### CODE REVIEW

File: src/resources/protobuf [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 636c52f4df519dd74000bbe8e42b8cb32ba8b51a
```
Comment (vmarchaud) [inline src/resources/protobuf:1]: nope nope nope, deleted this, we are sticking to a version of Protos.

**Comment by ramarro123** [comment]
ehm, i need a bit of help on that...

i have do a git checkout of the folder, with previous commit id to restore the folder, now
if i try to commit he say "nothing to commit" :(

any advice?

**Comment by vmarchaud** [comment]
@ramarro123 Sadly i'm not a git expert too, i will advise you to remake a PR ^^

**Comment by ramarro123** [comment]
seems better:)

**Pull request closed without merging**